releases/release-2.1-rc.4.md

```
@@ -14,7 +14,7 @@ summary: TiDB 2.1 RC4は2018年10月23日にリリースされ、安定性、SQL
     -   `UnionAll`演算子の結果が場合によっては正しくない問題を修正[＃8007](https://github.com/pingcap/tidb/pull/8007)
 -   SQL実行エンジン
     -   `AVG`関数の精度の問題を修正 [＃7874](https://github.com/pingcap/tidb/pull/7874)
-    -   `EXPLAIN ANALYZE`ステートメントを使用して、クエリ実行プロセス中の各演算子の実行時間と返された行数を含む実行時統計をチェックする機能をサポートします[＃7925](https://github.com/pingcap/tidb/pull/7925)
+    -   `EXPLAIN ANALYZE`ステートメントを使用して、クエリ実行プロセス中の各オペレーターの実行時間と返された行数を含む実行時統計をチェックする機能をサポートします[＃7925](https://github.com/pingcap/tidb/pull/7925)
```

markdownlint-cli2 (0.23.2)

[warning] Spaces after list markers
Expected: 1; Actual: 3

(MD030, list-marker-space)